build.sbt

```diff
@@ -261,7 +250,8 @@ lazy val ammRepl = project
     name := "ammonite-repl",
     libraryDependencies ++= Seq(
       "jline" % "jline" % "2.14.3",
-      "com.github.javaparser" % "javaparser-core" % "3.2.5"
+      "com.github.javaparser" % "javaparser-core" % "3.2.5",
+      "com.github.scopt" %% "scopt" % "3.5.0" % Test
```

Comment:
What's the reason for this addition?

Reply:
scopt is used in [`SourceTests`](https://github.com/lihaoyi/Ammonite/blob/73a874173cd337f953a3edc9fb8cb96556638fdd/amm/src/test/scala/ammonite/unit/SourceTests.scala#L52) (moved to the `repl` module), and is not around in the `repl` module without that.